packages/plugin-ext/src/hosted/node/hosted-instance-manager.ts

```diff
@@ -244,9 +245,9 @@ export abstract class AbstractHostedInstanceManager implements HostedInstanceMan
         if (environment.electron.is()) {
             command = ['yarn', 'theia', 'start'];
         } else {
-            command = processArguments.filter(arg => {
-                // remove --port= argument if set
-                if (arg.startsWith('--port=')) {
+            command = processArguments.filter((arg, index, args) => {
+                // remove --port=X and --port X arguments if set
+                if (arg.startsWith('--port') || args[index - 1] === '--port') {
```

Comment:
thx for the fix. But it seems only this part is modified but there are extra 100 changes that are not related to the fix (wrong format of code), could you please cleanup your PR ?

Reply:
Sorry, didn't notice. I will fix it.

Reply:
@benoitf now the formatting is the same as before